src/python/grpcio/README.rst

```diff
@@ -51,7 +51,7 @@ package named :code:`python-dev`).
   $ git submodule update --init
 
   # For the next two commands do `sudo pip install` if you get permission-denied errors
-  $ pip install -rrequirements.txt
+  $ pip install -r requirements.txt
```

Comment:
It works with both forms(with or without space between `-r` and `requirements.txt`).

So, it's unnecessary to make this change.